assets/src/edit-story/components/library/panes/media/media3p/api/fetch.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import https from 'https';
+
+/**
+ * Simple utility function to make GET requests.
+ *
+ * Used to fetch data from the Google Fonts API.
```

Comment:
> Used to fetch data from the Google Fonts API.
This comment is out of date.

Is it possible to just use the version of fetch in bin/utils/fetch.js?

Reply:
This is wrong either way. This util is used for a node.js context. In the browser, just use window.fetch